### PR TITLE
fix: #id [19305] correct window.options in seed components

### PR DESCRIPTION
**Pull request by kriswest**
[Kanban link](https://chartiq.kanbanize.com/ctrl_board/48/cards/19305/details/)

**Fix Welcome component and other example configs**
* The seed project contained some config options that were not working as they were not in the correct place (e.g. window.autoShow, which should be window.options.autoShow). This misleads clients all the time - particularly as we don't document window.options (we used to rely on OpenFin for this).

**Description of testing**
1. Run the seed project and check the Welcome component is working as normal.
1. Change some of the window.options.* settings (e.g. set `window.options.resizable = false` or `window.options.alwaysOnTop = true`) 
1. Restart Finsemble and see that the specified settings apply to the component correctly.

### CODE REVIEW

File: configs/application/components.json [hunk 1]
```diff
@@ -6,8 +6,12 @@
 				"url": "$applicationRoot/components/welcome/welcome.html",
 				"affinity": "workspaceComponents",
 				"frame": false,
-				"resizable": true,
-				"autoShow": true,
+				"options": {
+					"resizable": true,
+					"autoShow": true,
+					"backgroundColor": "#22262F",
```
Comment (NinjaManatee) [inline configs/application/components.json]: @kriswest Why did you add a background color?

Comment (kriswest) [reply configs/application/components.json]: To stimulate discussion of what we should actually be putting here. We've had dead options in place for over a year and at present have no docs on what options are available. This is one that has been requested in the past - although it doesn't actually prevent a white flash as the component loads annoyingly. 

I'll be following up on the ticket shortly as we should be discussing which options to surface in this example (which is likely to be one of the prime ways developers know about options, whether or not we document them).

Comment (kriswest) [reply configs/application/components.json]: As this doesn't actually prevent that white flash, I think it can be dropped. 

Is there anything else (in finsemble\src\common\FinsembleWindowData.d.ts) that you think would be useful to add into an explicit example @NinjaManatee?

Comment (kriswest) [inline configs/application/components.json]:
```suggestion
```

File: configs/application/components.json [hunk 2]
```diff
@@ -6,8 +6,12 @@
 				"url": "$applicationRoot/components/welcome/welcome.html",
 				"affinity": "workspaceComponents",
 				"frame": false,
```
Comment (kriswest) [inline configs/application/components.json]:
```suggestion
```

File: configs/application/components.json [hunk 3]
```diff
@@ -50,8 +53,10 @@
 				"url": "$applicationRoot/components/nonConfiguredComponent/nonConfiguredComponent.html",
 				"affinity": "systemComponents",
 				"frame": false,
```
Comment (kriswest) [inline configs/application/components.json]:
```suggestion
```
